app/models/host_status/execution_status.rb

```diff
@@ -37,6 +41,47 @@ def to_label(options = {})
     end
   end
 
+  class ExecutionTaskStatusMapper
+    attr_accessor :task
+
+    def self.sql_conditions_for(status)
+      status = STATUS_NAMES.key(status) if status.is_a?(String)
+
+      case status
+        when OK
+          [ "state = 'stopped' AND (result = 'success' OR result = 'warning')" ]
```

Comment:
I don't think it's a good idea on this level, if the job succeeded (no matter if someone had to help it) you want to find at among successful finished ones, if you really care about details, you can go to foreman task. Maybe we could add a warning icon next to status linking to the task.